awsauth.go

```diff
@@ -261,6 +261,7 @@ func GetCredentials(c *Config) (*awsCredentials.Credentials, error) {
 	}
 
 	if c.DebugLogging {
+                awsConfig.CredentialsChainVerboseErrors = aws.Bool(true)
```

Comment:
Looks like a formatting issue.

```suggestion
		awsConfig.CredentialsChainVerboseErrors = aws.Bool(true)
```

Reply:
My bad! Fixed.